ContactDetailsApi/Startup.cs

```diff
@@ -37,6 +36,7 @@
 using System.IO;
 using System.Linq;
 using System.Reflection;
+using Amazon.XRay.Recorder.Handlers.AwsSdk;
```

Comment:
Is there anything within this file using the namespace?

Reply:
Yep, this line below:
AWSSDKHandler.RegisterXRayForAllServices();

The IDE marks it as 'unused' because that line is disabled in debug (as we don't want xray running locally)

Reply:
Yep, this line below:
`AWSSDKHandler.RegisterXRayForAllServices();`

The IDE marks it as 'unused' because that line is disabled in debug (as we don't want xray running locally)